### ISSUE TITLE
First review! -> for GF

### ISSUE THREAD

**Issue by emmamarichal** [jrgdrs/Wittgenstein#8]
Hello @jrgdrs!

I'm Emma, and I'm in charge of the Wittgenstein onboarding.

I took a first look to the font, here are my comment/remarks:

### **Variable fonts:**
We would appreciate to onboard it as a variable font, since the two masters are compatible. However, we can't export instances that have been extrapolate (there is no ExtraLight master). I tried to generate a ExtraLight master from the instance, but it creates some interpolations issues, especially with open corners (we can't accept that, the export will crash).

<img width="681" alt="Screenshot 2024-05-29 at 13 56 29" src="https://github.com/jrgdrs/Wittgenstein/assets/64773544/cc129427-04e3-4ae1-b562-4c3e997a8eb9">

**So, what we can do here:**
- onboard it now, first with a weight axis from Regular to Black and add the other instance/master later. 
We can still let the extralight and light instance as statics, in the repo (but not in GF)
- wait until you add a compatible ExtraLight master

### **Other**
- five.osf.tosf -> five.tosf (to be applied to all numbers concerned)
- **🔥 FAIL Ensure dotted circle glyph is present and can attach marks:** you have to add `ogo` anchor to the dottedCircle
- There is an issue of interpolation in `degree` -> see `Interpolation.pdf` in `QA.zip`
- There are some `inferior` and `.inferior` -> `.inferior` and `.superior` are not reachable by any features. 
-> `.numr` `.dnom` `superior` `inferior` should be suffisant.

____

**Complete QA:**
[QA.zip](https://github.com/jrgdrs/Wittgenstein/files/15484102/QA.zip)

You will find here html proofs, the fontbakery report and the interpolation errors.

____

Let me know if you have any questions, and when these points are fixed. I'll then set up some last few things (build process, last QA, etc.) and send you a final PR from my fork :) 
Cheers!

**Comment by jrgdrs** [comment]
Hello @emmamarichal 

thank you for your onboarding support. I would be happy if you can continue the process and have carried out the fixes.

## EL
ExtraLight was a misunderstanding i think. I don't want to provide an EL cut, only the range from Regular (400) to Black (900). The EL was included in the Glyphs version, but I thought that i had switched off via the option so that it would not be exported. (When working on the interpolations, I like to display also the extrapolations, just as deviations and errors are much better visible. However, EL was not intended for exporting. That is why I have now removed the layer from Glyphs completely.

## Other
=> renamed the 10 tabular oldstyle figures from *.osf.tosf to *.tosf
=> added anchor for ogo and also for bottom to the dottedCircle (25cc)
=> for interpolation purposes corrected starting points of inner circle of the degree symbol (00b0) and did also some spacing correction but left the forms this bit sick
=> removed 20 glyphs named *.inferior and *.superior (with dot in name) and left *inferior and *superior (without dot in name) and adapted the feature tables accordingly

So i updated the repo and provided a new build named v1.44

Hope that works for you. Best Regards, Jörg

**Comment by kenmcd** [comment]
> So i updated the repo and provided a new build named v1.44

No variable fonts.
and 
No Medium (500).

Like the font!

**Comment by emmamarichal** [comment]
Thanks a lot @jrgdrs! :) I sent you a PR, with the fonts exported. I let you read my comment!

**Comment by jrgdrs** [comment]
Hello @emmamarichal,

did the merge but now the build exits with an error at building the variable font variants:

```
fontmake: Error: In 'master_ufo/Wittgenstein.designspace': Generating fonts from Designspace failed: 

Couldn't merge the fonts, because some values were different, but should have
been the same. This happened while performing the following operation:
GPOS.table.FeatureList.FeatureRecord[0].Feature.LookupCount

The problem is likely to be in Wittgenstein Black:
Expected to see .LookupCount==3, instead saw 2

```

Got the error also in the past, that's why I rolled back in the past and configured just without "buildVariable". I have no clue what's the reason. Tried also to use hints from [https://github.com/googlefonts/fontmake/issues/894](https://github.com/googlefonts/fontmake/issues/894) and equalized the zero-width-joiners with but without success. Do you have an idea or is it necessary for me to try an exercise as ufo investigator? 

Did also some spacing adaptions at the glyphs spacing after merging, so pls use the last version in repo.

Best Regards, Jörg

**Comment by jrgdrs** [comment]
Hello @emmamarichal,

Thank you, that's brilliant, the script has solved the knot. I have merged your pull request and deployed release 1.5. 

Best regards and have a nice weekend, Jörg
